python/lsst/pipe/tasks/makeGen3SkyMap.py

```diff
@@ -0,0 +1,131 @@
+# This file is part of pipe_tasks.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+import lsst.afw.geom as afwGeom
+import lsst.pex.config as pexConfig
+import lsst.pipe.base as pipeBase
+from lsst.daf.butler import Butler, DatasetType
+from lsst.skymap import skyMapRegistry
+
+
+class MakeGen3SkyMapConfig(pexConfig.Config):
+    """Config for MakeGen3SkyMapTask
+    """
+    datasetTypeName = pexConfig.Field(
+        doc="Name assigned to created skymap in butler registry",
+        dtype=str,
+        default="deepCoadd_skyMap",
+    )
+    name = pexConfig.Field(
+        doc="Name assigned to created skymap in butler registry",
+        dtype=str,
+        default=None,
+        optional=True
+    )
+    skyMap = skyMapRegistry.makeField(
+        doc="type of skyMap",
+        default="dodeca",
+    )
+    doWrite = pexConfig.Field(
+        doc="persist the skyMap? If False then run generates the sky map and returns it, "
+            "but does not save it to the data repository",
+        dtype=bool,
+        default=True,
+    )
```

Comment:
I don't think we need this option - if someone just wanted to create a SkyMap from a config, they wouldn't really need to use this task.